### PR TITLE
metaid: use int64 for 32-bit compatibility

### PR DESCRIPTION
**Pull request by NicolasDerumigny**
Fixes a resurgence of #5 following ec80bfbf.

### CODE REVIEW

File: pkg/metaid/ids.go [hunk 1]
```diff
@@ -34,21 +34,21 @@ func MakeWAPortalID(jid types.JID) networkid.PortalID {
...
 
 func MakeFBPortalID(portal int64) networkid.PortalID {
-	return networkid.PortalID(strconv.Itoa(int(portal)))
+	return networkid.PortalID(strconv.FormatInt(portal, 10))
 }
 
 func ParseUserID(user networkid.UserID) int64 {
-	i, _ := strconv.Atoi(string(user))
+	i, _ := strconv.ParseInt(string(user), 10, 64)
 	return int64(i)
```
Comment (tulir) [inline pkg/metaid/ids.go]: These casts are redundant now since `i` is already `int64` (also `ParseIDFromString`, `ParseUserLoginID` and `ParseFBPortalID`)

Comment (NicolasDerumigny) [reply pkg/metaid/ids.go]: I do not think so. Atoi returns a `int` (https://pkg.go.dev/strconv#Atoi) which is an `int32` on 32-bit arch, which is then converted to an `int64` in the return cast, but it is too late.

Comment (tulir) [reply pkg/metaid/ids.go]: I mean the casts, not the parsing

Comment (NicolasDerumigny) [reply pkg/metaid/ids.go]: Ow, my bad! Thanks, I'll correct that

**Review by tulir — APPROVED** [review]

**Pull request merged**